functions/displayBlock.js

```diff
@@ -0,0 +1,58 @@
+import * as Blocks from '@/components/blocks'
+import PropTypes from 'prop-types'
+
+/**
+ * Decide which block to display.
+ *
+ * @author WebDevStudios
+ * @param {object} block The block data.
+ */
+export default function displayBlock(block, index, siteSettings) {
```

Comment:
Missing `@param` declarations for `index` and `siteSettings`.

Reply:
@ravewebdev All fixed up. I also did some early destructuring.